### ISSUE TITLE
Search fails with SQL error

### ISSUE THREAD

**Issue by xtech86** [neicv/pagekit-search#3]
On a fresh install of pagekit I am getting:

```
An exception occurred while executing 'SELECT * FROM pk_docs_system_page a INNER JOIN pk_docs_system_node c ON (c.type LIKE ? AND c.data LIKE CONCAT ('%"defaults":{"id":', cast(a.id as char),'%')) WHERE ((c.status = ?) AND ((a.title LIKE ? OR a.content LIKE ?))) AND ((roles IS NULL) OR (roles REGEXP '(^|,)(2|3)($|,)')) GROUP BY a.title, a.content ORDER BY a.id DESC LIMIT 50 OFFSET 0' with params ["%page%", "1", "%css%", "%css%"]:

SQLSTATE[HY000]: General error: 1 no such function: CONCAT
```

When trying to search, the above search term was css. 

Many thanks

**Comment by neicv** [comment]
Hi! Do You use SQLite BD?
If Yes, in new release that bug will be fixed.
Or, you can use MySQL / MariaDB.

**Comment by neicv** [comment]
New release available. Plz check.

**Comment by neicv** [comment]
Closed.
that bug fixed in 0.1.5 rel.
